src/screens/GameDetails/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, useWindowDimensions } from 'react-native';
+import {
+  AspectRatio,
+  VStack as NativeBaseVStack,
+  Text,
+  HStack,
+  Fab,
+  useToast,
+  useTheme,
+} from 'native-base';
+import { useRoute } from '@react-navigation/native';
+import FastImage from 'react-native-fast-image';
+import RenderHtml from 'react-native-render-html';
+import { FloppyDisk } from 'phosphor-react-native';
+
+import ScrollView from '../../components/ScrollView';
+import Header from '../../components/Header';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import VStack from '../../components/VStack';
+import { useAppDispatch } from '../../store';
+import { setDrawerHeader } from '../../store/slices/navigation-slice';
+import rawg from '../../services/rawg.api';
+import { Game } from '../../interfaces/game.dto';
+import Loading from '../../components/Loading';
+
+type RouteParams = {
+  id: number;
+  slug: string;
+  name: string;
+};
+
+const GameDetails = () => {
+  const toast = useToast();
+  const { colors } = useTheme();
+  const dispatch = useAppDispatch();
+  const route = useRoute();
+  const { id, name } = route.params as RouteParams;
+  const { width } = useWindowDimensions();
+  const [game, setGame] = useState<Game>({} as Game);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    const getGame = async () => {
+      try {
+        const response = await rawg.get<Game>(
+          `https://api.rawg.io/api/games/${id}?key=e30c4b13ba264b8680f0fcab95f1b69a`,
```

Comment:
maybe create a constant for that API URL as well?

Reply:
I moved base url to rawg service, and api key to .env file